web/services/AI/ai_model.py

```
@@ -0,0 +1,190 @@
+    user_input = f"""
+    instructions: {AI_prompt_ai_quiz_corrector}
+    quiz_data: {quiz_data}
+    """
+
+    # Generate response
+    try:
+        response = model.generate_content(user_input)
+
+        json_match = re.search(r"\{.*\}", response.text, re.DOTALL)
+        if not json_match.group(0):
+            raise ValueError("No JSON object found in response")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)